test/invoice-workflow.e2e-spec.ts

```
@@ -0,0 +1,884 @@
+      const startWorkflowData = TestDataGenerator.generateStartWorkflowData();
+      startWorkflowData.customerId = customerId;
+
+      const scheduledWorkflowData =
+        TestDataGenerator.generateScheduledWorkflowData();
+      scheduledWorkflowData.customerId = customerId;
+
+      const recurringWorkflowData =
+        TestDataGenerator.generateRecurringWorkflowData();
+      recurringWorkflowData.customerId = customerId;
+
+      const startResponse = await request(app.getHttpServer())
```

ESLint

[error] 'startResponse' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)